app/components/primer/alpha/action_bar.pcss

```diff
@@ -0,0 +1,56 @@
+/* CSS for ActionBar  */
+
+.ActionBar {
+  display: flex;
+  align-items: center;
+  gap: var(--primer-controlStack-small-gap-condensed, 8px);
+}
+
+/* TODO: Replace with IconButton */
+.ActionBar-item .btn-octicon {
+  display: inline-flex;
+  align-items: center;
+  justify-content: center;
+  margin: 0;
+  padding: 0;
+  min-width: var(--primer-control-medium-size, 32px);
+  height: var(--primer-control-medium-size, 32px);
+  border-radius: var(--primer-borderRadius-medium, 6px);
+
+  &:hover {
+    color: var(--color-fg-default);
+    background-color: var(--color-segmented-control-button-hover-bg);
+  }
+
+  &:active {
+    background-color: var(--color-segmented-control-button-active-bg);
+  }
+}
+
+.ActionBar-divider {
+  margin: 0 var(--primer-control-medium-gap, 8px);
+  height: var(--primer-control-medium-lineBoxHeight, 16px);
+  border-left: var(--primer-borderWidth-thin, 1px) solid var(--color-border-subtle);
+}
+
+/* Variations */
+
+.ActionBar--condensed {
+  gap: 0;
+}
+
+.ActionBar--spacious {
+  gap: var(--primer-controlStack-small-gap-spacious, 16px);
+}
+
+.ActionBar--vertical {
```

Comment:
Added a `direction` option. Works ok so far:

<img width="74" alt="Screen Shot 2022-08-26 at 20 50 17" src="https://user-images.githubusercontent.com/378023/186897260-dde938bc-cc29-422f-82c3-579d1a809d16.png">

But yeah, it's kinda limited to `IconButton`. So we'd have to see what we wanna do for anything else.

Reply:
Revered this again (for now).